sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Microsoft.Azure.Management.SecurityCenter.csproj

```diff
@@ -6,10 +6,10 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.SecurityCenter</PackageId>
     <Description>Provides developers with libraries for the updated Azure Security Center platform under Azure Resource manager to view and manage security posture in and outside Azure.</Description>
-    <Version>3.0.0</Version>
+    <Version>3.0.1</Version>
     <AssemblyName>Microsoft.Azure.Management.SecurityCenter</AssemblyName>
     <PackageTags>management;security center;security;IoT security;</PackageTags>
-    <PackageReleaseNotes>Updated SDK client. Bumpes settings API Version</PackageReleaseNotes>
+    <PackageReleaseNotes>Updated SDK client.</PackageReleaseNotes>
```

Comment:
Add notes about all the breaking changes.

Reply:
Added. Can you please check if the comments are informative enough?